### ISSUE TITLE
Transfer Twitter login token?

### ISSUE THREAD

**Issue by burnsting** [adamjmcgrath/react-native-simple-auth#28]
I'm not sure if this is possible using this plugin, but I'm trying to log users in on my website using the access tokens (inside a react native app, obviously).

I got it working for Facebook. So I get a token from Facebook > use the same token in my Facebook PHP SDK and get the user's info.

But I can't get this to work with the twitteroauth PHP library I'm using. I tried a bunch of different ways, but in most cases you need a token and token_secret. Simple-auth only returns one token.

Should this be possible using the token I get?

Basically I'm trying to do $connection->get('account/verify_credentials'); in PHP with the token I get in JS, but it won't allow me..

**Comment by adamjmcgrath** [comment]
> in most cases you need a token and token_secret. Simple-auth only returns one token.

Yep - this is an issue with `react-native-simple-auth`, it should return you an additional `secret` token for Twitter. @ethul has a branch that supports this: https://github.com/adamjmcgrath/react-native-simple-auth/compare/master...ethul:topic/credentials I'll take a look at adding it this weekend. 

@ethul - feel free to raise a PR for that branch if you want.

**Comment by ethul** [comment]
PR #29 has been submitted.

**Comment by burnsting** [comment]
Thank you very much!

**Comment by adamjmcgrath** [comment]
Released 0.3.0
